core/base-service/coalesce-badge.js

```diff
@@ -149,9 +150,16 @@ export default function coalesceBadge(
     logoPosition = coalesce(overrideLogoPosition, serviceLogoPosition)
   }
   if (namedLogo) {
+    const iconSize = getIconSize(namedLogo.toLowerCase())
+
+    if (!logoWidth && iconSize && overrideLogoSize === 'cover') {
+      logoWidth = (iconSize.width / iconSize.height) * 14
```

Comment:
What is the significance of 14? Could this be a named constant?

Reply:
definitely want to avoid any new [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) :mage: